pkg/utils/gitops/controller.go

```diff
@@ -103,3 +105,47 @@ func (h *SuiteController) DeleteAllGitOpsDeploymentInASpecificNamespace(namespac
 		return len(gdList.Items) == 0, nil
 	}, timeout)
 }
+
+// CreateEnvironment creates a new environment
+func (h *SuiteController) CreateEnvironment(name, namespace string) (*appservice.Environment, error) {
+	environment := &appservice.Environment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: appservice.EnvironmentSpec{
+			Configuration: appservice.EnvironmentConfiguration{
+				Env: []appservice.EnvVarPair{
```

Comment:
We don't need the environment in this case

Reply:
I will change it to empty. Without it, it returns the error:`spec.configuration.env: Required value`.